### PR TITLE
SPUI-703 > Add first screen of profile details

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/app/shared/features/profile_details/view/profile_details.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          onBackButtonTap: () => Navigator.pop(context),
+        ),
+      ),
+      child: Column(
+        children: [
+          SProfileDetailsButton(
+            label: 'Email',
+            value: authInfo.email,
+            onTap: () {
+
```
Comment (eli1stark) [inline lib/app/shared/features/profile_details/view/profile_details.dart:28]: remove space here

File: plugins/simple_kit/lib/src/shared/components/simple_profile_details_button.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          alignment: Alignment.centerLeft,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Baseline(
+                baseline: 38.h,
+                baselineType: TextBaseline.alphabetic,
+                child: Text(
+                  label,
+                  style: sBodyText2Style.copyWith(color: SColorsLight().grey1),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/shared/components/simple_profile_details_button.dart:32]: comma

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/profile_details/view/profile_details.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/app/shared/features/profile_details/view/profile_details.dart:28]: done

File: plugins/simple_kit/lib/src/shared/components/simple_profile_details_button.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/shared/components/simple_profile_details_button.dart:32]: done

**Pull request merged**